CMakeLists.txt

```diff
@@ -407,16 +412,16 @@ if (ENABLE_JEMALLOC)
   endif()
 endif()
 
-find_package(LibIntl)
-if(LibIntl_FOUND)
-  include_directories(SYSTEM ${LibIntl_INCLUDE_DIRS})
-endif()
-
 find_package(Iconv)
 if(Iconv_FOUND)
   include_directories(SYSTEM ${Iconv_INCLUDE_DIRS})
 endif()
 
+find_package(LibIntl)
+if(LibIntl_FOUND)
+  include_directories(SYSTEM ${LibIntl_INCLUDE_DIRS})
+endif()
```

Comment:
Why was this moved?

Reply:
Because intl depends on iconv, findlibintl.cmake will use libIconv, so do this.